spring-xd-related-work.tex

```diff
@@ -1,2 +1,99 @@
 \section{Related Work}
 This section compares Spring XD to similar projects.
+
+\subsection{Spring XD and Spark}
+Spark is a general purpose framework for for large-scale data processing.
+
+For customers who need a rock solid ingestion platform based on mature open source technology, analytical stream APIs for the 80 perecentage use-case, RESTful control over stream processing and batch jobs across a range of big data technologies, as well as simplifying development and testing of Spark applications, Spring XD is available and supported today.
+
+Spring XD can do the following which Spark Streaming can't do.
+
+\begin{itemize}
+\item Reliably operate 24/7.
+\item Ingest data into HDFS following to best practices (no small files) and with zero code.
+\item Microbatching based on event count.
+\item One event at a time processing.
+\item Specify specific hosts where calculations are performed.
+\item Decouple infrastructure code from business logic/analytics.
+\end{itemize}
+
+It can do the following which Spark Batch processing can't do.
+
+\begin{itemize}
+\item Provider a REST API and lifecycle management of Spark jobs.
+\item Integrate with other Batch systems.
+\end{itemize}
+
+In the future Spring XD will continue to provide additional integration with Spark features such as MLLib and SparkSQL.
+
+\subsection{Spring XD and Storm}
+Storm is a distributed computation system for real-time stream processing.
+
+For customers who are in need of real-time stream processing, distributed data computations, and ETL capabilities, Spring XD as an unified runtime supports variety of use-cases ranging from classic enterprise to Big Data and IoT. 
+
+Spring XD can do the following which Storm can't do.
+
+\begin{itemize}
+\item Spring XD’s Shell compared with Storm’s API programming-paradigm delivers immediate productivity. No coding, IDE, bundling or packaging necessary. The high-level DSL abstracts complexities through developer-friendly fixtures.
```

Comment:
The '’' is a control character and causing LaTex to not format it properly.  Change apostrophe in the XD’s to Spring XD's.   This is a carry over from copying a Google doc to a Latex.tex file.